evm/vm/forks/sharding/headers.py

```diff
@@ -0,0 +1,14 @@
+from evm.constants import (
+    EMPTY_SHA3,
+)
+from evm.vm.forks.byzantium.headers import (
+    create_byzantium_header_from_parent,
+)
+
+
+def create_sharding_header_from_parent(parent_header, **header_params):
+    if 'transaction_root' not in header_params:
+        header_params['transaction_root'] = EMPTY_SHA3
```

Comment:
these can be changed *either* to `header_params.setdefault('transaction_root', EMPTY_SHA3)`

**or**

You can explicitly add them to the function signature.

```
def create_sharding_header_from_parent(parent_header, transaction_root=EMPTY_SHA3, ..., **header_params):
    ...
```

I think I like the `setdefault` approach best.